.github/CODEOWNERS

```diff
@@ -1,15 +1,15 @@
 /cray/modules/ars       @Cray-HPE/cray-management-systems
 /cray/modules/artifacts @Cray-HPE/cray-management-systems
 /cray/modules/badger    @Cray-HPE/hardware-management
-/cray/modules/bos       @Cray-HPE/cms-core-bos
+/cray/modules/bos       @Cray-HPE/cray-management-systems
 /cray/modules/bss       @Cray-HPE/hardware-management
 /cray/modules/capmc     @Cray-HPE/hardware-management
-/cray/modules/cfs       @Cray-HPE/cms-core-cfs
+/cray/modules/cfs       @Cray-HPE/cray-management-systems
 /cray/modules/config    @Cray-HPE/cloud
 /cray/modules/cps       @Cray-HPE/cray-management-systems
```

Comment:
This isn't owned by CMS. I don't know that any of our teams really owns it, so it should probably be moved to that bottom section with the other product-specific stuff